add 'Update on Coronavirus and SR' (#44)
srobo/team-emails: SR2020/2020-03-05-coronavirus.md

---
to: sr2020-teams
subject: Update on Coronavirus and SR
---

# Update on SR and the COVID-19 Coronavirus

Hello everyone, 

> TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.

Reviewer: I wonder whether this line is needed? It's a pretty short email as it is, and if anything it's a good idea for teams to read the whole thing anyway

Reviewer 2: Yeah, don't put a TL;DR in a team facing e-mail. We want them to be reading the whole e-mail! Informally between volunteers is one thing, externally facing is another.

> I'm sure you've heard about the threat caused by the COVID-19 coronavirus. and you've likely heard of many large events being cancelled due to concerns of spreading of the virus.

Reviewer 2: I don't feel "threat" is a good word to use in this context

Reviewer: Perhaps something like:

```suggestion
I'm sure you've heard about the COVID-19 coronavirus, and about many large events being cancelled due to concerns of spreading the virus.
```

> At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.

Reviewer: Change intro and bold

```suggestion
At this time, we're **not** intending to cancel the Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
```

Reviewer: > We are also working on a back-up plan should the event be unable to run.

This is news to me? And something which I doubt is necessary.

Reviewer 3: I would suggest removing the backup plan line

> At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.

Reviewer 2: I don't this paragraph is a good thing to say, the only thing the teachers will get out of it is that we may well cancel, and therefore they might pull out to save on hassle.

Reviewer: I disagree. Most teams seem pretty comfortable with the fact that we'd only cancel if we had to, and by that point things would be pretty bad.

Reassurance that we're following PHE also can't hurt.

Reviewer 2: Ok, perhaps I should have commented on sentences separately. I agree with the 2nd sentence of this, not the ones either side. Having regularly dealt with several team based events each year, whenever a comment around not cancelling is made normally I see teams fall away rather than feel assured.

I feel that this paragraph should instead read:
"We're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. Considering this is an event for minors, we will be taking decisions with an abundance of caution."

> With an abundance of caution, we've decided to take action during the competition to minimise risk. We will frequently disinfect any touch points (i.e. door handles & surfaces) during the competition, and will try to source hand sanitizer for use during the event, (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygene practises. We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].

Reviewer: Opening this paragraph with the same words as the previous one ended sounds strange.

```suggestion
To further prevent the risk of spread, we will frequently disinfect any touch points (i.e. door handles & surfaces) during the competition, and will try to source additional hand sanitizer for use during the event (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygene practises. We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].
```

Reviewer: > We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].

As discussed in Slack, I don't think this section is necessary

> With an abundance of caution, we've decided to take action during the competition to minimise risk. We will frequently disinfect any touch points (i.e. door handles & surfaces) during the competition, and will try to source hand sanitizer for use during the event, (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygene practises. We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].

Reviewer 2: The advice currently provided by the government says nothing about regularly santising surfaces, and recommends soap and water with hand sanitiser as a fall back. Soap and water should be readily available in the toilet facilities. Do not forget that, as a public venue, Reading Uni will have to be taking relevant precautions for all events at their venue anyway!

Additionally, and more crucially, the competition is still 6 weeks away, the advice could change hugely in that time. I **strongly** recommend not committing us to any specific actions. Just say that we will keep ourselves informed of the current situation, and take any appropriate actions.

Reviewer 3: I think it's fine to go more general as Scarzy says. The main point is for teams to know we are actively thinking about it.

Reviewer: Maybe just something like "We're following the recommended advise" is enough, rather than specifics.

Author: 'following recommended advice' is the bare minimum here. We should do more to ensure the event is safe and to settle the worries of the teams.

I've left in the above section but have clarified that it may change (therefore not committing us to any of it)

Author: I'm outvoted, so I'll remove it

> If you or your school has any requirements or questions about our handling of the event, don't hesitate to contact us at teams@studentrobotics.org.

Reviewer: ```suggestion
If you or your school has any additional requirements or questions about our handling of the event, please don't hesitate to contact us at teams@studentrobotics.org.
```

> If you or your school has any requirements or questions about our handling of the event, don't hesitate to contact us at teams@studentrobotics.org.

Reviewer 2: I wouldn't invite them to add requirements, as that'll encourage them to think of some. Encourage questions, and if their school does happen to have additional requirements they will ask about them.

```suggestion
If you or your school has any questions about our handling of the event, please don't hesitate to contact us at teams@studentrobotics.org.
```

---
to: sr2020-teams

> subject: Update on Coronavirus and SR

Reviewer 4: nit: I'm not sure how well known it is that "SR" == "Student Robotics" outside of volunteers. I suggest for clarity we expand the acronym, at least in the subject line.
```suggestion
subject: Update on Coronavirus and Student Robotics
```

---

# Update on SR and the COVID-19 Coronavirus

Hello everyone, 

I'm sure you've heard about the COVID-19 coronavirus, and about many large events being cancelled due to concerns of spreading the virus.

We're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. Considering this is an event for minors, we will be taking decisions with an abundance of caution.

> The competition is still a few weeks away and our plans will adapt to suit the situation at the time, however we are currently intending to heighten hygiene levels during the competition. For example, by frequently disinfecting any touch points (i.e. door handles & surfaces) during the competition, and we will try to source additional hand sanitizer for use during the event (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygiene practises.

Reviewer: There's some grammar oddities in this sentence. 

I also don't think it's necessary to mention increasing hygiene, and simply mentioning that we'll make sure people follow best practice is probably enough.

```suggestion
The competition is still several weeks away and our plans will adapt to suit the situation as things develop. We'll also be briefing all volunteers and attendees about best hygiene practices, and ensuring they're followed throughout the competition.
```

Reviewer 3: This is my main concern with the original

> We expect all teams who are attending the competition to follow the [official guidelines][phe-guidelines] on COVID-19 from Public Health England.

Reviewer: Is the word 'official' here necessary?

```suggestion
We expect all teams who are attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
```

If you or your school has any questions about our handling of the event, please don't hesitate to contact us at teams@studentrobotics.org.

[uk-coronavirus-page]: https://www.gov.uk/guidance/coronavirus-covid-19-information-for-the-public

> [returning-travellers]: https://www.gov.uk/guidance/coronavirus-covid-19-information-for-the-public#returning-travellers

Reviewer: This link doesn't appear to be referenced anymore, so we can remove it.
